app/src/main/java/pl/ludek/smat/home/pool_office_client/presentation/MainActivity.kt

```diff
@@ -25,105 +23,98 @@ class MainActivity : AppCompatActivity(), View.OnClickListener {
+            if( binding.relayOneOn.text == relayFirstOnStr){
+                dataFromRelay.postOneRelayOn().observe(this, Observer {
+                    if(!it.errorRelay){ binding.relayOneOn.text = relayFirstOffStr
                     }else{
                         showToast(errorRelayStr)
                     }
                 })
             }else {
                 dataFromRelay.postOneRelayOff().observe(this, Observer {
-                    if(!it.errorRelay){relayFirstOnButton.text = relayFirstOnStr
+                    if(!it.errorRelay){binding.relayOneOn.text = relayFirstOnStr
                     }else{
                         showToast(errorRelayStr)
                     }
                 })
             }
-
-            R.id.relayFieveOn -> if(relayFiveOnButton.text == relayFiveOnStr){
+        }
+        binding.relayFieveOn.setOnClickListener {
+            if(binding.relayFieveOn.text == relayFiveOnStr){
                 dataFromRelay.postFiveRelayOn().observe(this, Observer {
                     if (!it.errorRelay){
-                        relayFiveOnButton.text = relayFiveOffStr
+                        binding.relayFieveOn.text = relayFiveOffStr
                     }else{
                         showToast(errorRelayStr)
                     }
                 })
             }else{
                 dataFromRelay.postFiveRelayOff().observe(this, Observer {
-                    if(!it.errorRelay){relayFiveOnButton.text = relayFiveOnStr
+                    if(!it.errorRelay){binding.relayFieveOn.text = relayFiveOnStr
                     }else{
                         showToast(errorRelayStr)
                     }
                 })
             }
-
-            R.id.relayAllOn -> if(relayAllOnButton.text == relayAllOnStr){
+        }
+        binding.relayAllOn.setOnClickListener {
+            if(binding.relayAllOn.text == relayAllOnStr){
                 dataFromRelay.postAllRelayOn().observe(this, Observer {
                     if (!it.errorRelay){
-                        relayAllOnButton.text = relayAllOffStr
+                        binding.relayAllOn.text = relayAllOffStr
                     }else{
                         showToast(errorRelayStr)
                     }
                 })
             }else{
                 dataFromRelay.postAllRelayOff().observe(this, Observer {
-                    if(!it.errorRelay){relayAllOnButton.text = relayAllOnStr
+                    if(!it.errorRelay){binding.relayAllOn.text = relayAllOnStr
                     }else{
                         showToast(errorRelayStr)
                     }
                 })
             }
         }
+        setContentView(binding.root)
+    }
+
+    private fun setDataToLiveData(){
+         dataFromRepository.getDataFromLocalStorage().observe(this, Observer {
+             dataFromSensor.setPoolInfoData(it)
+         })
+     }
+
+    private fun initView(){
+        setContentView(R.layout.activity_main)
```

Comment:
This line has to be removed!
The content view is being set by `setContentView(binding.root)`